src/js/game_logic.js

```diff
@@ -44,21 +47,32 @@ const correctAnswer = () => {
         countElement.className = 'rhyme_count'
         guessElement.className = ''
         if (Game.currentCard.id < puzzles.length) {
-                loadCard(puzzles[Game.currentCard.id])
+            loadCard(puzzles[Game.currentCard.id])
         }
         else gameOver()
     }, 800)
 }
 
 const incorrectAnswer = () => {
+
     cardBlock.className += ' incorrect'
     countElement.className += ' incorrect'
     guessElement.className += 'incorrect'
+    numOfincorrectanswers++
+
+
     setTimeout(() => {
         cardBlock.className = 'rhyme_card'
         countElement.className = 'rhyme_count'
         guessElement.className = ''
     }, 500)
+    const wronganswers = document.getElementById('incorrect')
+    console.log(Game.currentCard.hint)
+    wronganswers.textContent = 'Number of incorrect: ' + numOfincorrectanswers
```

Comment:
Looks good! Let's make the phrase a little neater. How about 'Incorrect Answers: ' or 'Wrong Guesses: '?